plugins/woocommerce/src/Internal/StockNotifications/Admin/NotificationsPage.php

```diff
@@ -0,0 +1,107 @@
+<?php
+
+declare( strict_types = 1 );
+
+namespace Automattic\WooCommerce\Internal\StockNotifications\Admin;
+
+use Automattic\WooCommerce\Internal\StockNotifications\Admin\NotificationsListTable;
+use Automattic\WooCommerce\Internal\StockNotifications\Notification;
+
+/**
+ * Notifications admin page for Customer Stock Notifications.
+ */
+class NotificationsPage {
+
+	/**
+	 * Page URL.
+	 *
+	 * @const PAGE_URL
+	 */
+	const PAGE_URL = 'admin.php?page=wc-customer-stock-notifications';
+
+	/**
+	 * Init.
+	 *
+	 * @internal
+	 *
+	 * @return void
+	 */
+	final public function init() {
+		// Select section.
+		$section = '';
+
+		// Nonce is checked in NotificationsPage::delete and NotificationsPage::output just displays the page.
+		if ( isset( $_GET['section'] ) ) { // phpcs:ignore WordPress.Security.NonceVerification.Recommended
```

Comment:
Is there a way to make this a bit more semantic? Why is a "section" a delete action? maybe use a "action" param?

Reply:
This is a carry-over from the extension -- you are right that it can be improved. Done: https://github.com/woocommerce/woocommerce/pull/58356/commits/3a7a4d6b9916d5681fa1a00c4ed0406469b9a898